packages/@aws-cdk/pipelines/README.md

```diff
@@ -764,6 +766,21 @@ new pipelines.CodeBuildStep('Synth', {
   rolePolicyStatements: [
     new iam.PolicyStatement({ /* ... */ }),
   ],
+
+  // Control logging options
+  logging: {
+    cloudWatch: {
+      logGroup: logGroup,
+      prefix: 'prefix',
+      enabled: true,
+    },
+    s3: {
+      encrypted: true,
+      bucket: bucket,
+      prefix: 's3prefix',
+      enabled: true,
+    },
+  },
```

Comment:
I'm wondering if we can make this a little less manual for the user. For instance, if they're adding `cloudwatch` logging, I'd assume that they want `enabled: true`. Can we abstract some of this away? Maybe with a class similar to `Schedule` in `aws-events` that uses an enum like class. What do you think?

Reply:
I agree about providing human-friendly interfaces in high-level constructs. However, it would let users lose control if we overpack the functions. Users are able to set options separately for CloudWatch & S3 logging in `aws-codebuild` API, but not for `Schedule` in `aws-events` API.

I prefer to keep the current solution. But in the future, if any contributor requests a similar feature, we can design the interface.

Reply:
:+1: to @TheRealAmazonKendra's suggestion. Users will still be able to maintain full control. We could potentially do a separate property: eg, `cloudwatchLogging`, `s3Logging`

Reply:
Before abstracting the logging setting, I'd like to confirm the expected result when the user doesn't assign the object to CloudWatch logging. For now, CloudWatch logging is still enabled and saves the log to the default pattern.

Reply:
Let's leave the behavior as is if the user doesn't provide any input. I suppose we could have something like Logging.NONE() for logs to be disabled altogether, but we should leave the default alone so we don't unintentionally break customers code that might rely on it.